composer.json

```diff
@@ -18,7 +18,7 @@
     ],
     "require": {
         "php": ">=5.3.2",
-        "composer/composer": "1.*@dev",
+        "composer/composer": "1.0.x-dev",
```

Comment:
I agree. The problem lies in that composer/composer doesn't have anything but alpha release tags, and one of Sculpin's dependencies (component/bootstrap) needs code newer than alpha9.